src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Esfa.Recruit.Provider.Web.Configuration.Routing
+{
+    public static class RouteNames
+    {
+        public const string Dashboard_Index_Get = "Dashboard_Index_Get";
+        public const string Dashboard_ChangePassword = "Dashboard_ChangePassword";
+        public const string Dashboard_ChangeEmail = "Dashboard_ChangeEmail";
+
+        public const string DismissOutageMessage_Post = "DismissOutageMessage_Post";
```

Comment:
I think you might have removed the `CookieManagerController` which handles this route.

Reply:
👍